### ISSUE TITLE
Run and fine-tune personal SLM

### ISSUE THREAD

**Issue by vantastics** [Break-Through-Tech/Automation-Anywhere-1B-domain-specific-theme-labeling-via-slm-distillation#5]
Each team member run and fine tune a model. Focus on the getting one SLM to a score of more than 4 or 4.5 or even higher.

1. What is the difference in performance between the non-tuned (base) SLMs?
2. Can we really squeeze out better performance from the 360M model? - adding more epochs and learning rate, quantization etc.
3. Stretch goal for the week: Resolve RAM/VRAM issue on 7B, 8B, 9B models.

Van - Llama 3.2
Aashish - Llama3.1, Mistral 7B
Ancheng - Gemma 2, Gemma 3
 Wiesmes Gwen 2.5-1.5B,
Juliania Phi 3.5 mini
Kelly - Gwen 2.5 
Naavya -

**Comment by aaayushhhh** [comment]
Hi Van,
I believe I may have been assigned this issue by mistake. I am not supposed
to be part of this project, so I wanted to check whether this assignment
was intentional.
Thanks!

Best,
Ayush

**Comment by wiesmes** [comment]
This is the link to my results for fine tuning Quen

https://github.com/Break-Through-Tech/Automation-Anywhere-1B-domain-specific-theme-labeling-via-slm-distillation/blob/BTT_WWA_TEST/experiments/RESULTS.md
